src/Npgsql/PublicAPI.Shipped.txt

```diff
@@ -258,7 +258,6 @@ Npgsql.BackendMessages.FieldDescription.TypeModifier.set -> void
 Npgsql.BackendMessages.FieldDescription.TypeSize.get -> short
 Npgsql.BackendMessages.FieldDescription.TypeSize.set -> void
 Npgsql.INpgsqlDatabaseInfoFactory
-Npgsql.INpgsqlDatabaseInfoFactory.Load(Npgsql.NpgsqlConnection! conn, Npgsql.Util.NpgsqlTimeout timeout, bool async) -> System.Threading.Tasks.Task<Npgsql.NpgsqlDatabaseInfo?>!
```

Comment:
We haven't shipped the removal of this API yet :) So this change belongs in the Unshipped file exactly like the additions, prefixed by `*REMOVED*`.

Reply:
Ah, I see.